frontend/src/service-worker.ts

```diff
@@ -0,0 +1,80 @@
+/// <reference lib="webworker" />
```

Comment:
Is this autogenerated / should I understand this?

Reply:
Yep this is autogenerated by `create-react-app` - not sure what it does specifically but service-worker is for the app to work offline